manim/__init__.py

```diff
@@ -1,9 +1,16 @@
 #!/usr/bin/env python
 from __future__ import annotations
 
-from importlib.metadata import version
+from importlib.metadata import PackageNotFoundError, version
 
-__version__ = version(__name__)
+# Use installed distribution version if available; otherwise fall back to a
+# sensible default so that importing from a source checkout works without an
+# editable install (pip install -e .).
+try:
+    __version__ = version(__name__)
+except PackageNotFoundError:
+    # Package is not installed; provide a fallback version string.
+    __version__ = "0.0.0+dev"
```

Comment:
I'd go for something that is semantically somewhat more correct (it is unclear whether the version in this case is a development version, or whether someone has just forgotten to install the package). What about this:

```suggestion
    __version__ = "0.0.0+unknown"
```